debug.properties.default

```diff
@@ -1,2 +0,0 @@
-crashlytics.apikey=0000000000000000000000000000000000000000
```

Comment:
sorry, I deleted this by mistake, could you please restore the file with the one in master?